docs/kubectl_patch.md

```diff
@@ -25,7 +25,7 @@ kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}'
 
 ```
   -h, --help=false: help for patch
-  -p, --patch="": The patch to be applied to the resource JSON file.
+  -p, --patch="": The patch to be appied to the resource JSON file.
```

Comment:
I won't have time to circle back to this today (on-call) but the goal was
not to change anything substantive like hostnames (or screenshots of
commands referencing those hostnames).

On Mon, Jul 13, 2015 at 9:35 AM, Eric Tune <notifications@github.com> wrote:

> In docs/kubectl_patch.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11130#discussion_r34482332>
> :
>
> > @@ -25,7 +25,7 @@ kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}'
> >
> >  ```
> >    -h, --help=false: help for patch
> > -  -p, --patch="": The patch to be applied to the resource JSON file.
> > +  -p, --patch="": The patch to be appied to the resource JSON file.
>
> accidental deletion of character.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11130/files#r34482332>
> .
>